src/crypto/sha256_sse4.cpp

```diff
@@ -1020,15 +1020,15 @@ void Transform(uint32_t* s, const unsigned char* chunk, size_t blocks)
 ; This code schedules 1 blocks at a time, with 4 lanes per block
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-%define	MOVDQ movdqu ;; assume buffers not aligned 
+%define MOVDQ movdqu ;; assume buffers not aligned
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;; Define Macros
 
 ; addm [mem], reg
 ; Add reg to mem using reg-mem add and store
 %macro addm 2
-    add	%2, %1
-    mov	%1, %2
+    add %2, %1
```

Comment:
nit: given that we're fixing non-trailing tabs as well, the only remaining .cpp/.h file that still contains tabs is https://github.com/bitcoin/bitcoin/blob/75a5c8258ec5309fe506438aa3815608430b53d6/src/util/subprocess.h#L1149, if you edit again, consider including that

Reply:
Sure. https://github.com/arun11299/cpp-subprocess/pull/121.